mvp.feature

```diff
@@ -208,278 +208,277 @@ Feature: User with no Author identity in the Authority Register for Personas see
     When they click the link Ny <role> in the page Brukeradministrasjon
     Then the page adds a line with the fields
-        | ID | 
-        | Navn |
-	And a button Legg til that is enabled
-        Examples:
-            | Role |
-            | Institusjonsadmin |
-            | Institusjonskurator |
-            | Redaktør |
-            | Registrator |
-            | Applikasjonsadministrator |
-            | Bruker |
+      | ID   |
+      | Navn |
+    And a button Legg til that is enabled
+    Examples:
+      | Role                      |
+      | Institusjonsadmin         |
+      | Institusjonskurator       |
+      | Redaktør                  |
+      | Registrator               |
+      | Applikasjonsadministrator |
+      | Bruker                    |
 
   # Menuitems for Redaktør
-  Scenario: The user opens Redaktøroppsett
+  Scenario: 41. The user opens Redaktøroppsett
     Given the user is logged in as Redaktør
     When they click the menu item Redaktøroppsett
     Then the page Redaktøroppsett is opened
-	And has the fields
-        | Epost | 
-	And a button Lagre that is enabled
- 
+    And has the fields
+      | Epost |
+    And a button Lagre that is enabled
+
   # Menuitems for Applikasjonsadmin
-  Scenario: The user opens Institusjoner
+  Scenario: 42. The user opens Institusjoner
     Given the user is logged in as Applikasjonsadmin
     When they click the menu item Institusjoner
     Then the page Institusjoner is opened
-	And the user see a liste of all institution (customers)
-	And has the fields
-        | Institusjon | 
-        | Oprettet |
-        | Redaktør |
-	And a button Åpne that is enabled for each institution
+    And the user see a liste of all institution (customers)
+    And has the fields
+      | Institusjon |
+      | Oprettet    |
+      | Redaktør    |
+    And a button Åpne that is enabled for each institution
     And a button Opprett institusjon that is enabled
 
   # Actions from page : Institusjon (Add/Update)
-  Scenario: Applikasjonsadmin changes a new institution
+  Scenario: 43. Applikasjonsadmin changes a new institution
     Given the user is logged in as Applikasjonsadmin
     When they click the link Opprett institusjon or Åpne in the page Institusjoner
     Then the page Institusjon is opened with the fields
-        | Navn i organisasjonsregisteret | 
-        | Visningsnavn på institusjonen |
-        | Forkortet navn på institusjonen |
-        | Arkivnavn |
-        | CNAME |
-        | Institusjonsadmin ID |
-        | Feide Organisasjons ID |
-	And a button Legg til that is enabled
- 
\ No newline at end of file
+      | Navn i organisasjonsregisteret  |
+      | Visningsnavn på institusjonen   |
+      | Forkortet navn på institusjonen |
+      | Arkivnavn                       |
+      | CNAME                           |
+      | Institusjonsadmin ID            |
+      | Feide Organisasjons ID          |
+    And a button Legg til that is enabled
```

Comment:
To forskjellige scenarier? Et scenario der man legger til en ny institusjon og et scenario der man endrer en eksisterende institusjon? Hva skjer når man trykker på `Legg til`? Hvor havner brukeren?